zebrad/src/commands/start.rs

```diff
@@ -55,7 +55,8 @@ impl StartCmd {
         let state = ServiceBuilder::new().buffer(20).service(state_service);
 
         info!("initializing verifiers");
-        let verifier = zebra_consensus::chain::init(
+        // TODO: use the mempool verifier to verify mempool transactions (#2637, #2606)
+        let (chain_verifier, _mempool_verifier) = zebra_consensus::chain::init(
```

Comment:
Just so that there's no confusion that there is a different mempool transaction verifier type, it's the same type/instance used inside the block verifier:

```suggestion
        // TODO: use the transaction verifier to verify mempool transactions (#2637, #2606)
        let (chain_verifier, _tx_verifier) = zebra_consensus::chain::init(
```